source/ID/concordium-did.rst

```diff
@@ -0,0 +1,661 @@
+
+  ccd-sc-ref = "/" entrypoint *1("/" parameter) *1("?" "standard" "=" standard)
+  parameter = base16char
+  entrypoint = 1*(ALPHA / DIGIT / punctuation)
+  standard = "CIS" "-" 1*(DIGIT)
+  punctuation = "!" / DQUOTE / "#" / "$" / "%" / "&" / "'" / "(" /
+                ")" / "*" / "+" / "," / "-" / "." / "/" / ";" /
+                "<" / "=" / ">" / "?" / "@" / "[" / "\" / "]" /
+                "^" / "_" / "`" / "{" / "|" / "}" / "~"
+
+See the details about *dereferencing* a smart contract instance reference in :ref:`sci-reference`.
+
+Examples
+--------
+
+An account on testnet:
+
+``did:ccd:testnet:acc:3ZFGxLtnUUSJGW2WqjMh1DDjxyq5rnytCwkSqxFTpsWSFdQnNn``
+
+
+A Concordium credential on mainnet
+
+``did:ccd:mainnet:cred:9aa3641a212da36a9ffae6e6085b9cf486ca9b44fa059aa74565b0a1c0f7052d8e71168beccf299d767f3961b33aaae2``
+
+A smart contract instance on the default network (``mainnet``):
+
+``did:ccd:sci:12:0``
+
+A public key:
+
+``did:ccd:pkc:0c7f4421e44a4385850b883e3bbf098f5a9853ef6f1a862c2ce2856381b5f5e3``
+
+A smart contract instance with the ``issuerKeys`` entrypoint that does not take any parameters
+
+``did:ccd:sci:321/issuerKeys``
+
+A smart contract instance with the ``credentialEntry`` entrypoint taking a parameter
+
+``did:ccd:sci:123/credentialEntry/ee763364dc1a47d6aa4cc6bdb005e2b2``
+
+
+Concordium DID Documents
+========================
+
+Account DID
+-----------
+
+The goal of the Account DID Document is to provide information about the account authentication data, including a possibility to reference particular pieces of data, such as public keys.
+In order to do that, it specifies a `DID verification method <did-vefication-method_>`_ that reflects the account authentication data: public keys grouped into credentials.
+
+The Account DID Document MUST contain the following data:
+
+- ``id`` - the DID of the account.
+- ``verificationMethod`` - the account's verification method.
+  It is a nested :ref:`threshold scheme <concordium-did-verification-method>` requiring at ``T`` out of ``M`` credentials to sign; each credential uses its own threshold scheme requiring ``R_i`` out of ``N_i`` keys to sign, where ``i = 1..M``.and ``j = 1..N_i``.
+  The credentials are identified by a `DID fragment`_ ``#credential-i``, and the keys in each credentials by ``#key-j-i`` where ``i = 1..M`` and ``j = 1..N_i``.
+- ``authentication`` - authentication method for the account.
+
+The document MAY include any other public data of a Concordium account.
+
+.. note::
+
+  A `DID fragment`_ allows for referencing a particular credential, or a key in the Account DID Document.
+  The fragment is used to locate the (unique) JSON object by matching the DID URL with the object's ``id`` property.
+
+.. seealso::
+
+  `Dereferencing a DID URL`_ in the W3C Credentials Community Group draft report.
+
+
+.. code-block:: json
+
+  {
+    "id": "did:ccd:NET:acc:ADDR",
+    "verificationMethod": [
+      {
+        "id": "did:ccd:NET:acc:ADDR#acc-1",
```

Comment:
The fragment `#acc-1` is used to identify a verification method for the account. It could be any string; `acc-1` is confusing, so I'll change it to, e.g. `acc-vm`.  It is used as a relative reference in `authentication`. That is, this verification method is used to authenticate operations for the account.

Reply:
Fiexd here and in the credential DID document as well.